src/Core/View.js

```diff
@@ -458,4 +458,49 @@ View.prototype.execFrameRequesters = function execFrameRequesters(when, dt, upda
     }
 };
 
+const _eventCoords = new Vector2();
+/**
+ * Extract window coordinates from a mouse-event
+ * @param {mouseevent} event
+ * @return {THREE.Vector2} - window coordinates (in pixels, 0-0 = top-left)
+ */
+View.prototype.eventToWindowCoords = function eventToWindowCoords(event) {
```

Comment:
may be change to `mouseEventToViewerCoords`, replace `event` by `mouseEvent` and `windows` by `viewer`

Reply:
It could be a touchevent as well. So I think `event` is fine.

Window coordinates could be named View coordinates but Window Coordinates is the usual name... I can clarify the documentation and say they are relative to the View (e.g: `0-0 = top-left corner of the View`).

Reply:
ok, then function should indicate, in his parameters, which finger is used?
Documentation can be clarify

Reply:
I updated the documentation.